autorest/utility.go

```diff
@@ -138,11 +138,31 @@ func MapToValues(m map[string]interface{}) url.Values {
 	return v
 }
 
+// ToStringSlice method converts interface{} to []string.
```

Comment:
There are qualifiers on this, I'd update the comment to make clear that the `interface{}` must have a particular shape. I.e. it must be a slice of a type derived from string.